packages/deploy/deploy/700_land/polygonLand/732_setup_polygonland_v3_royalty.ts

```diff
@@ -0,0 +1,45 @@
+import {DeployFunction} from 'hardhat-deploy/types';
+import {HardhatRuntimeEnvironment} from 'hardhat/types';
+import {DEPLOY_TAGS} from '../../../hardhat.config';
+
+const ROYALTY_PERCENTAGE = 500;
```

Comment:
What do you think about creating a file in the deploy package with like royaltiesConfig.ts and store all the royalties there named like:

LAND_ROYALTY_BPS

Reply:
I don't think that we need a global configuration. 
The royalties will be managed directly by interacting with the contract (most probably with defender proposals).
This scripts are helpers to set some values different from zero (mostly on testnet).

Reply:
The initial values for defender proposals do come from the scripts though. Kinda agree a global file would make it nice and obvious...